src/main/java/uk/gov/hmcts/reform/cpo/data/CasePaymentOrderEntity.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.cpo.data;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.Column;
+import javax.persistence.GenerationType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import java.util.Date;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "case_payment_orders")
+public class CasePaymentOrderEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "created_timestamp")
+    private Date createdTimestamp;
```

Comment:
Can consider also adding `@CreationTimestamp` to have it auto-populated when new entities are persisted.